client/web/src/search/input/SearchContextMenu.scss

```
@@ -49,42 +56,53 @@
             font-size: 0.75rem;
         }
 
-        &-default {
-            background-color: var(--color-bg-1);
-            color: var(--text-muted);
-            border: 1px solid var(--border-color-2);
-            font-size: (10.5 / 16) + rem;
-            line-height: 1rem;
-            padding: 0 0.375rem;
-            border-radius: 2px;
-            margin-left: 0.25rem;
-        }
-
         &:not(:active) {
             .search-context-menu__item-description {
                 color: var(--text-muted);
             }
         }
 
         &:hover,
-        &:active {
+        &:active,
+        &:focus {
             .search-context-menu__item-default {
-                border-color: transparent;
+                background-color: var(--color-bg-1);
+            }
+
+            .theme-redesign & {
+                background-color: var(--primary);
+                color: var(--light-text);
+
+                .search-context-menu__item-description {
+                    color: var(--light-text);
+                }
             }
         }
 
         &--selected {
             background-color: var(--color-bg-4);
+            .search-context-menu__item-default {
```

```suggestion
            > .search-context-menu__item-default {
```

Our [CSS style guide](https://docs.sourcegraph.com/dev/background-information/web/web_app#styles) says to avoid descendant / child selectors where unavoidable, and I think this is a valid exception, but we should prefer to use child selectors unless the general descendant selector is required.

---

Alternatively we could also apply a class with the `--selected` modifier directly on the badge.